GANDLF/entrypoints/anonymizer.py

```diff
@@ -1,12 +1,83 @@
 #!usr/bin/env python
 # -*- coding: utf-8 -*-
+import logging
+import os
+import argparse
+import sys
+import yaml
+from typing import Optional
+import click
+from deprecated import deprecated
 
-import os, argparse, sys, yaml
 from GANDLF.anonymize import run_anonymizer
 from GANDLF.cli import copyrightMessage
+from GANDLF.entrypoints import append_copyright_to_help
 
 
-def main():
+def _anonymize_images(
+    input_dir: str, output_file: str, config_path: Optional[str], modality: str
+):
+    input_dir = os.path.normpath(input_dir)
+    output_file = os.path.normpath(output_file)
+    # TODO: raise an error if config pass provided but not exist (user made a typo?)
+    if config_path and os.path.isfile(config_path):
+        config = yaml.safe_load(open(config_path, "r"))
+    else:
+        config = None
```

Comment:
This is just to improve code coverage

```suggestion
    config = None
    if config_path and os.path.isfile(config_path):
        config = yaml.safe_load(open(config_path, "r"))

    assert config is not None, "A valid config file should be passed"
```

Reply:
Are you sure we want to require config file from users?
Actually, according to the [anonymizer code](https://github.com/mlcommons/GaNDLF/blob/master/GANDLF/anonymize/__init__.py#L21) , it's ok if config is empty / not passed at all

Reply:
Got it, thanks for the reminder. In that case, I think the `assert` portion of my suggestion can be removed. That should be the same as before, right? 😄

Reply:
Right! Commited changes:)